src/detections/fetch/repository/get.js

```diff
@@ -0,0 +1,6 @@
+import { client } from 'startup/client'
```

Comment:
Let's reorder our imports in the following manner: 
```js
import { from } from 'rxjs'

import { client } from 'startup/client'
import { GET_DETECTIONS } from 'detections/fetch/respository/query'
```

Reply:
Done